data/lm/generate_lm.py

```diff
@@ -133,23 +156,31 @@ def main():
         description="Generate lm.binary and top-k vocab for DeepSpeech."
     )
     parser.add_argument(
-        "--input_txt",
-        help="Path to a file.txt or file.txt.gz with sample sentences",
+        "--input_txt", "-i",
+        help="Path to a file.txt or file.txt.gz with sample sentences. "
+             "Pass argument multiple times for multiple files.",
         type=str,
+        action='append',
         required=True,
     )
     parser.add_argument(
-        "--output_dir", help="Directory path for the output", type=str, required=True
+        "--output_dir", "-o",
```

Comment:
can you avoid reformatting in the PR, or at least make that in a different commit? it's making the PR harder to follow

Reply:
Yes, sorry,  I was thinking it might not have been wise to do both at once. I'll revert that side of things and leave it for later (if at all)

Reply:
Some of the reformatting was as I was trying to keep to guidance of 100 chars but I'll reduce the changes to make it easier

Reply:
I'm not against the reformatting, it's just that having it in a separate commit makes it easier to review actual changes from reformatting :)